yogstation/code/modules/spacepods/equipment.dm

```diff
@@ -0,0 +1,349 @@
+
+/obj/item/spacepod_equipment/cargo/large/ore/on_uninstall()
+	UnregisterSignal(spacepod, COMSIG_MOVABLE_MOVED)
+	..()
+
+/obj/item/spacepod_equipment/cargo/large/ore/proc/spacepod_moved(obj/spacepod/SP)
+	if(storage)
+		for(var/turf/T in SP.locs)
+			for(var/obj/item/stack/ore in T)
+				ore.forceMove(storage)
+
+/obj/item/spacepod_equipment/cargo/chair
+	name = "passenger seat"
+	desc = "A passenger seat for a spacepod."
+	icon_state = "sec_cargo_chair"
+	var/occupant_mod = 1
+
+/obj/item/spacepod_equipment/cargo/chair/on_install(obj/spacepod/SP)
+	..()
+	SP.max_passengers += occupant_mod
+
+/obj/item/spacepod_equipment/cargo/chair/on_uninstall()
+	spacepod.max_passengers -= occupant_mod
+	..()
+
+/obj/item/spacepod_equipment/cargo/chair/can_uninstall(mob/user)
+	if(spacepod.passengers.len > (spacepod.max_passengers - occupant_mod))
+		to_chat(user, "<span class='warning'>You can't remove an occupied seat! Remove the occupant first.</span>")
+		return FALSE
+	return ..()
+
+/*
+///////////////////////////////////////
+/////////Weapon System///////////////////
+///////////////////////////////////////
+*/
+
+/obj/item/spacepod_equipment/weaponry/disabler
+	name = "disabler system"
+	desc = "A weak disabler system for space pods, fires disabler beams."
+	icon_state = "weapon_taser"
+	projectile_type = /obj/item/projectile/beam/disabler
+	shot_cost = 400
+	fire_sound = 'sound/weapons/taser2.ogg'
+	overlay_icon = 'yogstation/icons/obj/spacepods/2x2.dmi'
+	overlay_icon_state = "pod_weapon_disabler"
+
+/obj/item/spacepod_equipment/weaponry/burst_disabler
+	name = "burst disabler system"
+	desc = "A weak disabler system for space pods, this one fires 3 at a time."
+	icon_state = "weapon_burst_taser"
+	projectile_type = /obj/item/projectile/beam/disabler
+	shot_cost = 1200
+	shots_per = 3
+	fire_sound = 'sound/weapons/taser2.ogg'
+	fire_delay = 30
+	overlay_icon = 'yogstation/icons/obj/spacepods/2x2.dmi'
+	overlay_icon_state = "pod_weapon_disabler"
+
+/obj/item/spacepod_equipment/weaponry/laser
+	name = "laser system"
+	desc = "A weak laser system for space pods, fires concentrated bursts of energy."
+	icon_state = "weapon_laser"
+	projectile_type = /obj/item/projectile/beam
+	shot_cost = 600
+	fire_sound = 'sound/weapons/Laser.ogg'
+	overlay_icon = 'yogstation/icons/obj/spacepods/2x2.dmi'
+	overlay_icon_state = "pod_weapon_laser"
+
+// MINING LASERS
+/obj/item/spacepod_equipment/weaponry/basic_pod_ka
+	name = "weak kinetic accelerator"
+	desc = "A weak kinetic accelerator for space pods, fires bursts of energy that cut through rock."
+	icon = 'yogstation/goon/icons/obj/spacepods/parts.dmi'
+	icon_state = "pod_taser"
+	projectile_type = /obj/item/projectile/kinetic/pod
+	shot_cost = 300
+	fire_delay = 14
+	fire_sound = 'sound/weapons/Kenetic_accel.ogg'
+
+/obj/item/spacepod_equipment/weaponry/pod_ka
+	name = "kinetic accelerator system"
+	desc = "A kinetic accelerator system for space pods, fires bursts of energy that cut through rock."
```

Comment:
so they're KA's except they fire lasers so they're not KAs?

Reply:
what?